packages/admin/cms-admin/src/documents/types.ts

```diff
@@ -47,3 +49,11 @@ export interface DocumentInterface<
     InfoTag?: React.ComponentType<{ page: PageTreePage }>;
     anchors: (input: DocumentInput) => string[];
 }
+
+export interface DependencyInterface<GQLQuery = Record<string, unknown>, GQLQueryVariables = OperationVariables> {
```

Comment:
There might be dependencies that are not documents, please move this type into the dependencies/ folder